File: Placements.DataAccess/Placement/Models/Course.cs
using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class Course
{
    public long Id { get; set; }

    public string? Name { get; set; }

    public string? FullForm { get; set; }

    public virtual ICollection<CompanyJobCourse> CompanyJobCourses { get; set; } = new List<CompanyJobCourse>();

    public virtual ICollection<Studentacademic> Studentacademics { get; set; } = new List<Studentacademic>();
}

File: web-app/src/app/services/types/Course.ts
import { CompanyJobCourse } from "./CompanyJobCourse";
import { Studentacademic } from "./Studentacademic";

export type Course = {
  Id: number;
  Name: string;
  FullForm?: string;
  Studentacademics: Studentacademic[];
  CompanyJobCourses: CompanyJobCourse[];
};
